### PR TITLE
You can no longer use batons if you have chunky fingers.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

You can no longer use batons if you have chunky fingers.

## Why It's Good For The Game

hulk + baton is stupid as shit, try punching people with hulk for once

## Changelog

:cl:
balance: You can no longer use batons if you have chunky fingers.
/:cl:

### CODE REVIEW

**Comment by Redrover1760** [comment]
I love insuls nerfs for no reason.

**Comment by bruhlookatthisdood** [comment]
this makes no sense

**Comment by moocowswag** [comment]
It was my idea to nerf hulk and goof decided to make it chunky fingers rather than just hulk.

**Comment by Pickle-Coding** [comment]
> this makes no sense

A baton is a sophisticated weapon that is designed to incapacitate people.

A hulk is an angry green person that just wants to smash things, and would probably smash their baton if they attempted to use it.

An engineer wearing insulated gloves just can't get a proper grip for the baton to use effectively.

**Comment by SethLafuente** [comment]
This would make golems unable to use batons I see this as a good thing 

This will make people with insuls unable to use batons buffing combat gloves this is good

This makes hulks unable to use batons this is also a good thing

**Comment by moocowswag** [comment]
Okay so here is some general stuff that needs to be said about this pr though.

1. I think this is obvious, just make it so you can still harm intent with the baton , you just cant use the stun baton / instant knockdown baton attack.


2. >try punching people with hulk for once

this is what im really worried about, hulk punches are not used because their mechanic is not well known but as soon as stun batons are nerfed and people start using the punches...itl become obvious..

Hulk punches completely ignore armor and have a wound bonus of 20 which means only a couple hits will result in a fracture which is pretty much an instant round removal for an antag when it happens.

2.5 on the flip side, they can be blocked by just holding a chair in your hand, 50% block rate (about 75% when duel wielding.

**Comment by ATH1909** [comment]
how hard is it to hit people with a stick

**Comment by ATH1909** [comment]
also not a fan of the tedium/lame deaths this would invoke with insuls

and of golems getting fucked by proxy unless they aug themselves

**Comment by SethLafuente** [comment]
> how hard is it to hit people with a stick
> 
> 
> 
> like, telebatons are LITERALLY just sticks, you don't even have the excuse of "but they need to hold down a button!" with them


Stun batons is a stick with a charge of electric mechanism on the top we can say that the hulks don't have enough mental intelligence to properly hit targets with the mechanism correctly to stun them or that they put so much force I to the hit that the mechanism just breaks

**Comment by ATH1909** [comment]
alternative idea: hulks always harmbaton

you can check for TRAIT_HULK instead of TRAIT_CHUNKYFINGERS to avoid hitting golems, ashwalkers, and insulated gloves enjoyers as collateral, since they're too ANGRY to use finesse, not too clumsy.

File: code/game/objects/items/melee/baton.dm
Comment (Mothblocks) [inline code/game/objects/items/melee/baton.dm:137]: What happens if this is just `balloon_alert(`? Does it come from the in-hand item? If so that's a shame

**Comment by ATH1909** [comment]
> Stun batons is a stick with a charge of electric mechanism on the top we can say that the hulks don't have enough mental intelligence to properly hit targets with the mechanism correctly to stun them or that they put so much force I to the hit that the mechanism just breaks

they can hit people with the pointy end of a spear or the fiery end of a welder, I don't see how this is any different.

if the mechanism is supposed to break, make that actually affect the baton itself.

File: code/game/objects/items/melee/baton.dm
Comment (optimumtact) [reply code/game/objects/items/melee/baton.dm:137]: could target it from teh user instead

**Comment by SethLafuente** [comment]
> alternative idea: hulks always harmbaton
> 
> 
> 
> you can check for TRAIT_HULK instead of TRAIT_CHUNKYFINGERS to avoid hitting golems, ashwalkers, and insulated gloves enjoyers as collateral, since they're too ANGRY to use finesse, not too clumsy.

I actually think it would be good to remove golems from using stun barons same with ash walkers and insuls 

Golems gigantic force army  with near infinite numbers needs a nerf 

Ash walkers are suppose to be tribal primitive type of thing makes sense for them not to be able to use it

Insults would be a combat gloves buff plus insuls need some disadvantages in combat

**Comment by realthomas69** [comment]
I dont hate this but you seriously need to be more creative
like make it so that they break the baton into pieces and it shocks them or something when they try to use it

File: code/game/objects/items/melee/baton.dm
Comment (Mothblocks) [reply code/game/objects/items/melee/baton.dm:137]: then this should just be balloon_alert

**Comment by saprasam** [comment]
> I dont hate this but its pretty boring you need to be more creative like make it so that they break the baton into pieces and it shocks them or something when they try to use it

this

**Comment by SethLafuente** [comment]
> > I dont hate this but its pretty boring you need to be more creative like make it so that they break the baton into pieces and it shocks them or something when they try to use it
> 
> 
> 
> this
> 
> also once again messes over people wearing insuls


I think this is a good thing to see less people with insuls combat and stuff makes sense also a buff of combat gloves 


Why do you want to be able to use insuls and a baton?

File: code/game/objects/items/melee/baton.dm
Comment (Iamgoofball) [reply code/game/objects/items/melee/baton.dm:137]: based on my understanding of balloon_alert(), it comes from what it's called on and is shown to the person in arg1

**Comment by KittyNoodle** [comment]
please explain, how i, a heritic who doesn't want to get got by the randomly shocked grilles in maints, am supposed to get combat gloves

**Comment by Wallemations** [comment]
Wouldn't it make more sense for them to be unable to toggle the baton on & off?
I'm not opposed to the change overall I'm just confused over how having big fingers makes you unable to swing something

**Comment by JohnFulpWillard** [comment]
> please explain, how i, a heritic who doesn't want to get got by the randomly shocked grilles in maints, am supposed to get combat gloves

Don't wear insuls and simply don't walk into grilles?
Or don't use batons because you have your own stuns?

**Review by MrMelbert — CHANGES_REQUESTED** [review]
FYI this does not block harm+stun baton


https://user-images.githubusercontent.com/51863163/202052419-61b97e40-33a0-427e-8fc2-db97ce7f99e4.mp4

**Comment by Mothblocks** [comment]
why wouldnt you just turn it on then put on the gloves though

**Comment by Wallemations** [comment]
> why wouldnt you just turn it on then put on the gloves though

I just said that swinging it should turn it off before you hit the person

**Comment by saprasam** [comment]
> I think this is a good thing to see less people with insuls combat and stuff makes sense also a buff of combat gloves
> 
> Why do you want to be able to use insuls and a baton?

compared to ranged combat where you can realize rather quickly "oh i have insuls on oops" and take them off then go back to fighting in about 2 seconds, batong combat is way more dangerous & having insuls on will completely gut you in any altercation

**Comment by Mothblocks** [comment]
in that case its probably a good idea to apply to both guns and stun batons a balloon alert when you change your active hand to one that it wont work, like "your gloves make it hard to hold" or somethin

**Comment by optimumtact** [comment]
obviously stunbaton has one of those squeeze triggers you have to hold down to keep it active

**Comment by saprasam** [comment]
> obviously stunbaton has one of those squeeze triggers you have to hold down to keep it active

if this was the case somebody would have to code batons to turn them off once you put them in a pack or a belt because that's just not the case currently

**Comment by optimumtact** [comment]
>if this was the case somebody would have to code batons to turn them off once you put them in a pack or a belt because that's just not the case currently

putting it into your bag or belt wtihout locking it in the off position means it gets squeezed on

**Comment by moocowswag** [comment]
According to half life 2 wiki the stun baton is "shake to toggle charge"

**Comment by private-tristan** [comment]
> Or don't use batons because you have your own stuns?

heretic has 0 hardstuns outside of the blade heretic backstab ability, everything else is knockdowns, the only way to stamcrit an enemy as heretic is to wear mask of madness, wait 30 seconds infront of someone, and then use your stun arm on them to stamcrit them.

or you can just hand -> shove -> baton -> win (stamina combat is a meme)

**Comment by MrDoomBringer** [comment]
great PR 
now stop pussy footing around and give hulk the nodrop-romerol-zombie-hand-items but for smashing

**Comment by Troxonater** [comment]
Whats bad about hulk and baton? Giving yourself hulk doesn't instantly antag you as "angry man go kill people" like the real hulk when you first start having to control him, so obviously your character still has some level of control of themselves RP wise. Hulk already has a ton of tradeoffs, no Martial Arts, no guns, you're obviously a hulk if you have it (Even with my strategies to hide hulk, the first time you get hit by a stun baton people should be able to figure it out). Forcing antag hulks to basically round remove only because their kidnapping tools don't work doesn't seem like a good change to me, Also, the way to kill a hulk is already just keep your distance and shoot them, if you try to get close to a hulk you should be punished by whatever they have up their sleeve. Also, as melbert pointed out the collateral telekinesis nerf is dumb.

**Comment by moocowswag** [comment]
>Also, the way to kill a hulk is already just keep your distance and shoot them, if you try to get close to a hulk you should be punished by whatever they have up their sleeve.

Shoot them with what??? not everyone has a ranged weapon. 

I dont think you understand how out of proportion hulk is as a powerspike, I even see SEC AND COMMAND PLAYERS USING IT.

**Comment by Troxonater** [comment]
> > Also, the way to kill a hulk is already just keep your distance and shoot them, if you try to get close to a hulk you should be punished by whatever they have up their sleeve.
> 
> Shoot them with what??? not everyone has a ranged weapon.
> 
> I dont think you understand how out of proportion hulk is as a powerspike, I even see SEC AND COMMAND PLAYERS USING IT.

In the rules it says that non sec aren't supposed to be validhunting, if a hulk is being that big of a problem warden can open armory and distribute laser guns. A good portion of heads of staff start off with an energy gun. Also, to be fair I don't play as much as I used to but back when I was no lifeing manuel every other day I rarely saw sec and command use hulk. I view hulk as an alternate play style that is a gigantic middle finger to stun based combat and I'm ok with having it in its current state.

**Comment by Mothblocks** [comment]
imo we might be better off just making hulk uber powerful then turning it into an uplink item rather than nerf it every several months

@optimumtact imagine a geneticist final objective and you become a super big hulk like a heretic ascension

**Comment by Shadowflame909** [comment]
Every month is a new hulk Nerf but genetics still isn't getting new content because of hulk

Society moment

**Comment by improvedname** [comment]
> This would make golems unable to use batons I see this as a good thing

don't golems get a hardstun on their punches anyways?

**Comment by optimumtact** [comment]
not falling for ideas guys bait

**Comment by Mothblocks** [comment]
just musing just musing

**Comment by moocowswag** [comment]
This doesnt kill hulk itl still be a powerful mutation.

**Comment by Shadowflame909** [comment]
> This doesnt kill hulk itl still be a powerful mutation.

If I need Hulk I'm literally an antag who did their job to counter sec one hitting me and ending my round. (Most likely a heretic trying to sac the sec officer target.) If they can easily move out of the way and draw a laser or beat the crap out of one for the same damage they are taking, since they are wearing armor too.

Is it really the end game reward of this department if the only target it's necessary for can now easily counter it? 

I think not.  As maintainers ruled before, I say no more hulk nerfs unless genetics gets something equally cool. Or there is no benefit over doing the job that isn't more useful using another department, like chemistry or engi.

**Comment by SethLafuente** [comment]
> > This doesnt kill hulk itl still be a powerful mutation.
> 
> 
> 
> If I need Hulk I'm literally an antag who did their job to counter sec one hitting me and ending my round. (Most likely a heretic trying to sac the sec officer target.) If they can easily move out of the way and draw a laser or beat the crap out of one for the same damage they are taking, since they are wearing armor too.
> 
> 
> 
> Is it really the end game reward of this department if the only target it's necessary for can now easily counter it? 
> 
> 
> 
> I think not.  As maintainers ruled before, I say no more hulk nerfs unless genetics gets something equally cool. Or there is no benefit over doing the job that isn't more useful using another department, like chemistry or engi.

First technically this is a chunky fingers nerf and an indirect hulk nerf so I don't think that fits the policy probably does dosent matter 

Second hulk isn't end game genetics get real geneticist be getting this shit 12 minutes in hardly end game at all

Second this dosent make hulk any less strong then it actually is you take no Stam damage from batons or disabled you make a fuckton of damage even fracture you have plenty of things as a an antag that benefit with hulk too 

This is an overall good change to balance hulk out think about it 

You take no damage from batons or disablers?

That's a huge plus with the com of not being able to use those batons and disablers this is a fair trade and hulk is still plenty strong and useful


I assume the only reason you would be against this change is that you want to have your cheat gene to fill in for your lack of general robustness

**Comment by optimumtact** [comment]
This PR actually has very little to do with hulk and rests instead on two simple points

1) stun combat is not very fun outside of the paranoia it induces about people near you.
2) stun combat where one side is IMMUNE to the property of being stunned is absolutely, world ending, catastrophically unfun for literally everyone but the stun immune person.

**Comment by moocowswag** [comment]
> If I need Hulk I'm literally an antag who did their job to counter sec one hitting me and ending my round.

Okay but you took hulk to be immune to stun combat not be a god at it. You have other tools at your disposal, I`m gonna repeat myself, this pr being merged will very likely bring about other strong weapons hulks have at their disposal (their fracture causing, armor ignoring fists)

**Comment by Shadowflame909** [comment]
> > > This doesnt kill hulk itl still be a powerful mutation.

Hulk has gained a lot of downsides over the years. You lose your humanity and borgs/ai is free to crit you if your a threat, you used to be able to be disabled with a disabler, and the current status has it where you break your limbs upon breaking walls. I would say that it is already Balanced and has long left its powerful state.

> First technically this is a chunky fingers nerf and an indirect hulk nerf so I don't think that fits the policy probably does dosent matter

It's literally intended to be a hulk nerf as stated in the about PR text.

> Second hulk isn't end game genetics get real geneticist be getting this shit 12 minutes in hardly end game at all
Must be some really good RNG to solve radioactive and strength within the first 12 minutes.

> Third this dosent make hulk any less strong then it actually is you take no Stam damage from batons or disabled you make a fuckton of damage even fracture you have plenty of things as a an antag that benefit with hulk too

While good. This will put me in extended conflict with a sec officer, which no antagonist wants to be in because the longer they are, the easier it is to call for help. And unlike other crew, since hulk removes human status then they can easily also be taken out by borgs too. So that's also not good.

> Fourth this is an overall good change to balance hulk out think about it
> 
> You take no damage from batons or disablers?

Batons will still hurt though. Also thats literally the upside of hulk.

> That's a huge plus with the com of not being able to use those batons and disablers this is a fair trade and hulk is still plenty strong and useful

Yes that is the upside of hulk.

> And fifth I assume the only reason you would be against this change is that you want to have your cheat gene to fill in for your lack of general robustness and you are possibly getting mad people are noticing it's overall unfairness and they are make it balanced.

If me getting hulk and being able to counter sec's I win stick is unbalanced then I don't know what balanced is any more.

Stealth Antags lose here. Murderboners win again.

This PR dramatically reduces the upsides of getting hulk while keeping their plentiful amount of downsides. What good is locking myself to only being able to use melee weapons, if the best melee weapon in the game is now locked off from me?

**Comment by SethLafuente** [comment]
> > > > This doesnt kill hulk itl still be a powerful mutation.
> 
> 
> 
> Hulk has gained a lot of downsides over the years. You lose your humanity and borgs/ai is free to crit you if your a threat, you used to be able to be disabled with a disabler, and the current status has it where you break your limbs upon breaking walls. I would say that it is already Balanced and has long left its powerful state.
> 
> 
> 
> > First technically this is a chunky fingers nerf and an indirect hulk nerf so I don't think that fits the policy probably does dosent matter
> 
> 
> 
> It's literally intended to be a hulk nerf as stated in the about PR text.
> 
> 
> 
> > Second hulk isn't end game genetics get real geneticist be getting this shit 12 minutes in hardly end game at all
> 
> Must be some really good RNG to solve radioactive and strength within the first 12 minutes.
> 
> 
> 
> > Third this dosent make hulk any less strong then it actually is you take no Stam damage from batons or disabled you make a fuckton of damage even fracture you have plenty of things as a an antag that benefit with hulk too
> 
> 
> 
> While good. This will put me in extended conflict with a sec officer, which no antagonist wants to be in because the longer they are, the easier it is to call for help. And unlike other crew, since hulk removes human status then they can easily also be taken out by borgs too. So that's also not good.
> 
> 
> 
> > Fourth this is an overall good change to balance hulk out think about it
> 
> > 
> 
> > You take no damage from batons or disablers?
> 
> 
> 
> Batons will still hurt though. Also thats literally the upside of hulk.
> 
> 
> 
> > That's a huge plus with the com of not being able to use those batons and disablers this is a fair trade and hulk is still plenty strong and useful
> 
> 
> 
> Yes that is the upside of hulk.
> 
> 
> 
> > And fifth I assume the only reason you would be against this change is that you want to have your cheat gene to fill in for your lack of general robustness and you are possibly getting mad people are noticing it's overall unfairness and they are make it balanced.
> 
> 
> 
> If me getting hulk and being able to counter sec's I win stick is unbalanced then I don't know what balanced is any more.
> 
> 
> 
> Stealth Antags lose here. Murderboners win again.
> 
> 
> 
> This PR dramatically reduces the upsides of getting hulk while keeping their plentiful amount of downsides. What good is locking myself to only being able to use melee weapons, if the best melee weapon in the game is now locked off from me?

Will just tackle two points 

First of all don't try to put on oh the AI and borgs no longer human this is hilarious to hear have you ever heard of moths, lizards and cat people they don't have the abilities you do and yet they are not human and they are doing fine. 

Just use a flash honestly this isn't even a point that should be artful about 

And second you being a hulk already counters sec officers win stick since they can't stun you what's your point picking up a stun baton isn't countering shit you are just becoming unfairly strong and as oranges said about stun combat where one party is immune it sucks for everyone but the one that is stun inmune

**Comment by Shadowflame909** [comment]
> Okay but you took hulk to be immune to stun combat not be a god at it. You have other tools at your disposal, I`m gonna repeat myself, this pr being merged will very likely bring about other strong weapons hulks have at their disposal (their fracture causing, armor ignoring fists)

Losing the ability to use lasers forces you into a lot closer fights then you otherwise would ever get in. No time to heal if your playing whack a mole with the sec officer every second until one of you drops. 

Look at it like this. This PR get's merged, your a heretic genetist who gets hulk and has a sec officer target. You lead them somewhere quiet and start punching them to death. They shout for help and start harmbatonning you as well. By the time you down them, your health is at least orange, backup is coming your way and you're now put in the situation where you're going loud even when you didnt want to.

No stun baton hulk is asking for it to only be used by those who going loud was a non issue (the robust) and once more stops being a tool to close the game between the unrobust and sec.

**Comment by SethLafuente** [comment]
> > Okay but you took hulk to be immune to stun combat not be a god at it. You have other tools at your disposal, I`m gonna repeat myself, this pr being merged will very likely bring about other strong weapons hulks have at their disposal (their fracture causing, armor ignoring fists)
> 
> 
> 
> Losing the ability to use lasers forces you into a lot closer fights then you otherwise would ever get in. No time to heal if your playing whack a mole with the sec officer every second until one of you drops. 
> 
> 
> 
> Look at it like this. This PR get's merged, your a heretic genetist who gets hulk and has a sec officer target. You lead them somewhere quiet and start punching them to death. They shout for help and start harmbatonning you as well. By the time you down them, your health is at least orange, backup is coming your way and you're now put in the situation where you're going loud even when you didnt want to.
> 
> 
> 
> No stun baton hulk is asking for it to only be used by those who going loud was a non issue (the robust) and once more stops being a tool to close the game between the unrobust and sec.
> 
> 

Your scenario where you bring a sec officer to a secluded area and you get orange health while they call for help and batoning you let me deconstructed 


1.they won't be able to say their exact location while typing in combat most people don't do that since it will make him take at least an extra 2 punches or more even you only take 5 punches to down people 

2. If you double shove them they won't be able to attack you since they will still be shove stunned even more if you double grab them this can let you easily down them without them even hitting you if you possession yourself right 

3.There is more things to your disposal other than just punches this is more of a few tips from me you can use 

1.bolases. 

Normal bolas- easy to make slows officers down preventing them from running 

Reinforced bolas- knocks down people and their weapons allowing you to take them and making them an easy target since they are knocked down 

Energy bolas- 

Same as the normal bolas upside is that people can't catch it or have it to hit you with it if they take it off

Gondola bolas- rarer but easy to get if you explore to space absolutely broken takes around more than 1 minute to take off makes people pacifists unable to hit you

2.Throwing weapons

General throwing weapons such as spears, energy pens, stars and paper airplanes are great for dealing an initial or later hit from a distance without getting hit 

3.flashes

4.bottles 

There is more things you can use with hulk to assure a win unless they are more robust than you and you die which is part of the game 

I have more points about thismy point is that hulk dosent need stun baton as you say to kill security officers and such you have stun immunity heavy ass punches stun immunity isn't just stun batons you cannot be shoved or have any stam damage towards you you have no idea how much things you avoid you can't be knocked with xeno spit you can't be knocked down with cult hands too you are immune to so many things and yet you want more? This is just pure greed

**Comment by Shadowflame909** [comment]
This argument is getting pretty circular so I will leave you with this. This change will only prompt another hulk nerf later. Because the only people left for who it will be viable for, are those who can down a mass amount of players without it.

**Comment by moocowswag** [comment]
Hulk making you nonhuman isnt even a downside because you become the very best thing to fight against silicons...and xenos....and abductors.

Hulk being countered by lasers is a moot point because the same console you get hulk from also has a soft counter to lasers.

> Batons will still hurt though. Also that`s literally the upside of hulk.

A stun baton is 10 force, hulk punch is 15 armor ignoring with wound bonus of 20, that means the more armor you both are wearing the more it is in your favor not to mention youre likely to stack fractures.

**Comment by SethLafuente** [comment]
> This argument is getting pretty circular so I will leave you with this. This change will only prompt another hulk nerf later. Because the only people left for who it will be viable for, are those who can down a mass amount of players without it.

Other words for saying hulk isn't gona be broken so someone that isn't great at the game won't be able to win against someone that is better than them do to the unbalanced broken nature of hulks current state 

This is good shadow and I'm sure people will still use hulk I will use hulk when it becomes balanced and this seems like a PR that will do this

**Comment by SethLafuente** [comment]
Also hulks become human after being beaten to crit so borgs won't be able to fill kill them allne

**Review by MrMelbert — CHANGES_REQUESTED** [review]
- As was mentioned `balloon_alert` can come from the item itself
- Check for `is_holding` to fix TK memes

**Comment by Troxonater** [comment]
Everything Shadowflame909 says is correct. I have never seen anyone but coders complain about hulk, the vast majority of players don't even want to touch genetics because of how brain dead grinding genetics is. Lets go with mothblocks's decree, no more genetics nerfs until its actually fun to play for anything other than the barely powergaming genes that have a nerf implemented every 3 months. Hulk is not a problem, only coders think its unfair that hulk is a god at stun combat. And you know what? Hulk is a god at stun combat, which is fine. You know all those stun alternatives you mentioned? I'm like 99% sure bolas work on hulk, and they work even better on a hulk that's trying to rush you with a baton. The average antag has no way to get a stun baton reliably in the first place, other than killing beepsky which I've never even seen anyone else do. Why does security get a 1 hit win button against 99% the crew and the moment someone does that back to them "OH NO NERF IT!!!" even though after the first or second officer goes down armory should be wide open. This is one of the biggest skill issue PR's I have ever seen that's actually being taken seriously.

**Comment by moocowswag** [comment]
>Why does security get a 1 hit win button against 99% the crew and the moment someone

Thats the problem, the only reason im for this or even suggested it is because sec and command players started using hulk.

**Comment by Troxonater** [comment]
> > Why does security get a 1 hit win button against 99% the crew and the moment someone
> 
> Thats the problem, the only reason im for this or even suggested it is because sec and command players started using hulk.

If a sec or command manages to hide hulk, thats pretty impressive. So 99% of the time you will know if a sec or command has hulk just by looking at them. It's up to the player then (who usually has the surprise factor vs sec or command) to craft a bola, buy a gun, get adrenaline sacs as changleing, etc. This is a sandbox rp game. Stop locking things down.

**Comment by moocowswag** [comment]
> get adrenaline sacs as changleing,

adrenaline sacs dont actually do much against stun batons as ling youre better off either transform stinging the hulk into a non hulk or spam transforming into a monkey or back.

This is the problem, alot of antags dont have inherent access to ways to counter stun batons, the weapon is balanced around outplaying via shoves or slips or throws and hulks are immune to those.

**Comment by NamelessFairy** [comment]
I agree with the first half of thee why its good for the game. Although this PR touches more than just hulk, not too strong feelings about it but I would like to know why you see this as beneficial to the gameplay for other sources of chunky fingers.

**Pull request merged**
